### ISSUE TITLE
the test accuracy

### ISSUE THREAD

**Issue by CVsaber** [swathikirans/ego-rnn#3]
Hi @swathikirans
I trained the network in your way and then tested it in two stream approach, but in any case my accuracy could not reach 77.59% on the GTEA 61 dataset. My best result is 75%.So I want to ask how you trained? How to use RGB Model and Flow Model to train in two stream? 
Thank you!

**Comment by swathikirans** [comment]
Hi @DEepLiker, 

Run main-run-twoStream.py after training the RGB and Flow models separately. We reported the best performing model, after several runs, in the paper. Since GTEA61 is imbalanced and because of the stochastic nature of training, it is not guaranteed to obtain the exact same accuracy. I would suggest using EGTEA dataset since it is large enough to train on deep models.

**Comment by CVsaber** [comment]
Thanks a lot
when i run main-run-twoStream.py, I see i need to choose the RGB model and Flow model, I want to ask which model should I choose? the last epoch model or the val best model? or others?

**Comment by swathikirans** [comment]
Both the model from last epoch and best model should ideally give the best result since we are training down till the last layers of the individual streams.

**Comment by frostinassiky** [comment]
@DEepLiker When you run `main-run-twoStream.py`, do you change any parameters? 
I got only  71.55% for joint training. 
Now I am testing different hyper-parameters with the hope that the score can change - -

**Comment by chenwenhe11** [comment]
@frostinassiky I have the same problem, have you solved it？
